obsreport/obsreporttest/obsreporttest.go

```diff
@@ -79,6 +82,13 @@ func CheckExporterLogsViews(t *testing.T, exporter string, acceptedLogRecords, d
 	checkValueForView(t, exporterTags, droppedLogRecords, "exporter/send_failed_log_records")
 }
 
+// CheckExporterQueueRetryViews checks that for the current exported values for queue retry views match given values.
+// When this function is called it is required to also call SetupRecordedMetricsTest as first thing.
+func CheckExporterQueueRetryViews(t *testing.T, exporter string, queueLength int64) {
+	exporterTags := tagsForExporterView(exporter)
+	checkValueForProducer(t, exporterTags, queueLength, "exporter/queue_size")
+}
```

Comment:
For the moment move this in the queued_retry_test.go

Reply:
I don't want to have more public APIs :D

Reply:
moved :)